api-reference/sdk-reference/package-aqueduct/package-aqueduct.integrations/aqueduct.integrations.s3_integration.md

```diff
@@ -25,27 +25,39 @@ Class for S3 integration.
 #### file
 
 ```python
-def file(filepath: str,
+def file(filepaths: Union[List[str], str],
          format: S3FileFormat,
          name: Optional[str] = None,
          description: str = "") -> TableArtifact
 ```
 
-Retrieves a file from the S3 integration.
+Reads multiple files from the S3 integration into a single TableArtifact.
 
 **Arguments**:
 
-  filepath:
-  Filepath to retrieve from.
+  filepaths:
```

Comment:
Is this documentation reflected in the SDK? If not, we should update it there and autogenerate these docs.